samples/net/gsm_modem/boards/frdm_uart2_dts.overlay

```diff
@@ -6,4 +6,7 @@
 &uart2 {
 	status = "okay";
 	current-speed = <115200>;
+	gsm: gsm_modem {
```

Comment:
By convention node names should have `-` instead of `_`: `s/gsm_modem/gsm-modem/`

Reply:
You mean you prefer this to be:
```
gsm: gsm-modem {
```
?

Reply:
Yes, I prefer that.

Okay, I see that there is no consistency right now. If you look at https://docs.zephyrproject.org/latest/guides/dts/intro.html#syntax-and-structure all the nodes are with `-` instead of `_`. Additionally I remember the same convention (about using `-` in node names and `_` in node labels) was used in Linux dts files, at least some time ago.